### ISSUE TITLE
Cannot register two different instances to the same id

### ISSUE THREAD

**Issue by zepumph** [phetsims/capacitor-lab-basics#182]
After running phet-io tests on capacitor lab basics, I got this error. @andrewadare do you know anything about this?

```
Uncaught Error: Assertion failed: Cannot register two different instances to the same id: capacitorLabBasics.capacitanceScreen.model.circuit.switchedCapacitor.plateSizeProperty
Error: Assertion failed: Cannot register two different instances to the same id: capacitorLabBasics.capacitanceScreen.model.circuit.switchedCapacitor.plateSizeProperty
    at window.assertions.assertFunction (http://localhost:8080/assert/js/assert.js:21:13)
    at Object.addInstance (http://localhost:8080/phet-io/js/phetio.js?bust=1480964525046:386:19)
    at Object.addInstance (http://localhost:8080/phet-io/js/phetio.js?bust=1480964525046:58:14)
    at Tandem.addInstance (http://localhost:8080/tandem/js/Tandem.js?bust=1480964525046:87:36)
    at new Property (http://localhost:8080/axon/js/Property.js?bust=1480964525046:88:38)
    at SwitchedCapacitor.Capacitor [as constructor] (http://localhost:8080/capacitor-lab-basics/js/common/model/Capacitor.js?bust=1480964525046:67:30)
    at new SwitchedCapacitor (http://localhost:8080/capacitor-lab-basics/js/common/model/SwitchedCapacitor.js?bust=1480964525046:35:15)
    at createCircuitComponents (http://localhost:8080/capacitor-lab-basics/js/common/model/circuit/ParallelCircuit.js?bust=1480964525046:646:7)
    at CapacitanceCircuit.ParallelCircuit [as constructor] (http://localhost:8080/capacitor-lab-basics/js/common/model/circuit/ParallelCircuit.js?bust=1480964525046:109:30)
    at new CapacitanceCircuit (http://localhost:8080/capacitor-lab-basics/js/capacitance/model/CapacitanceCircuit.js?bust=1480964525046:39:21)
```

**Comment by andrewadare** [comment]
CL:B needed to be caught up for instrumentation to ArrowNode. I fixed it yesterday in fc8c84dea58529976917c1e63c2a772cdcd02b25 and a508b892197c3389f9e22d4489c5b99e365a027b. Tests with the aqua server pass in my working copy. @zepumph can you confirm, and close if all is well for you? Thanks.

**Comment by zepumph** [comment]
Looks good to me, and the tests are passing! Closing.
